### ISSUE TITLE
`eslint --init` failed

### ISSUE THREAD

**Issue by jinliming2** [eslint/eslint#11862]
**Tell us about your environment**

* **ESLint Version: 5.16.0**
* **Node Version: v10.16.0**
* **npm Version: 6.9.0**

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```bash
$ npx eslint --init
```
```
? How would you like to use ESLint? To check syntax, find problems, and enforce code style
? What type of modules does your project use? JavaScript modules (import/export)
? Which framework does your project use? React
? Where does your code run? (Press <space> to select, <a> to toggle all, <i> to invert selection)Browser
? How would you like to define a style for your project? Answer questions about your style
? What format do you want your config file to be in? JavaScript
? What style of indentation do you use? (Use arrow keys)
❯ Tabs
  Spaces
```

**What did you expect to happen?**

ESLint inits successfully.

**What actually happened? Please include the actual, raw output from ESLint.**

No more response when I wanna choose `Spaces`. The whole terminal was just dead. Even I press control+c, it doesn't respond.

**Are you willing to submit a pull request to fix this bug?**

No idea.

**Comment by ranupratapsingh** [comment]
I am also the same problem. It gets stuck at last ask. Tried different options
* (like using airbnb style guide or JSON vs Javascript format config file)
* using yarn or npm package managers.

Facing issue on both ubuntu linux(18.04), node v10.15.3, eslint v5.16.0
and macbook(mojave), node v12.1.0, eslint v5.16.0

```
npx eslint --init
? How would you like to use ESLint? To check syntax, find problems, and enforce code style
? What type of modules does your project use? JavaScript modules (import/export)
? Which framework does your project use? React
? Where does your code run? (Press <space> to select, <a> to toggle all, <i> to invert selection)Browser
? How would you like to define a style for your project? Use a popular style guide
? Which style guide do you want to follow? Airbnb (https://github.com/airbnb/javascript)
? What format do you want your config file to be in? JavaScript
Checking peerDependencies of eslint-config-airbnb@latest
Local ESLint installation not found.
The config that you've selected requires the following dependencies:

eslint-plugin-react@^7.11.0 eslint-config-airbnb@latest eslint@^4.19.1 || ^5.3.0 eslint-plugin-import@^2.14.0 eslint-plugin-jsx-a11y@^6.1.1
? Would you like to install them now with npm? (Y/n) 

```

**Comment by mysticatea** [comment]
Thank you for your report.

I confirmed it on my PC.

Looks like it can hang up since v5.5.0. (v5.4.0 worked fine)

**Comment by mysticatea** [comment]
https://github.com/eslint/eslint/compare/v5.4.0...v5.5.0

Hmm, updating `inquirer` may be the cause...

**Comment by mysticatea** [comment]
Ah... OK. Looks like a regression of [inquirer 6.4.0](https://github.com/SBoudrias/Inquirer.js/releases/tag/inquirer%406.4.0). If you install `inquirer@6.3` manually, `eslint --init` works fine.

**Comment by lehainam-dev** [comment]
I experienced the same error. 
Stuck at `Would you like to install them now with npm? (Y/n)` (nothing happen when press keys, even `Ctrl + C`).

- OS: MacOS Mojave
- ESLint Version: 5.16.0
- Node Version: v10.15.3
- npm Version: 6.9.0

**Comment by gosukiwi** [comment]
Same issues, can confirm it works for me with `5.4` on Win10.

**Comment by jacobashymo** [comment]
@lehainam-dev I'm having the same issue. All my specs are the same except for the Node version - I'm running v12.4.0

**Comment by shardool** [comment]
![eslintCrash](https://user-images.githubusercontent.com/5981515/59889778-dd56de00-9382-11e9-8f4f-df0afc7a8a7e.gif)

**Comment by mysticatea** [comment]
Hi. This has been identified as an upstream bug: https://github.com/SBoudrias/Inquirer.js/issues/811

Please wait for the upstream fix.

A workaround, for now, is to install the previous version of `inquirer` manually:

```
npm install inquirer@6.3
```

**Comment by lowest-fi-ever** [comment]
@mysticatea still doesn't work (after downgrade to inquirer@6.3)

**Comment by mysticatea** [comment]
SBoudrias/Inquirer.js#811 looks fixed. Please re-install `inquirer`.

**Comment by jacobashymo** [comment]
It's not fixed unless I'm doing something wrong
I reinstalled inquirer at 6.3 and it still freezes up the terminal

**Comment by lehainam-dev** [comment]
@jacobashymo It fixed. Try to remove `node_modules` and `package-lock.json` (or `yarn.lock`) and reinstall modules.

**Comment by jacobashymo** [comment]
![Screen Shot 2019-06-24 at 10 27 52 AM](https://user-images.githubusercontent.com/41447792/60031674-cb1ebd80-966a-11e9-9d35-6091323c03f1.png)
Still does this

**Comment by lehainam-dev** [comment]
@jacobashymo As the commands say, I think you've installed `eslint` globally in your machine. One more thing is now the issue is fixed, you don't need to install a specific version of `inquirer` to make `eslint` working. Just redo the remove part, then:
`yarn install`
`npx eslint --init` (notice the `npx` part)

**Comment by jacobashymo** [comment]
It still did the same thing
I did a `yarn install`, `npm install`, and `npm install --save inquirer@latest`, and got the same end
